Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -415,16 +432,19 @@ def check_if_fetch_incidents_is_tested(missing_ids, integration_ids, id_set, con
     return missing_ids, tests_set
 
 
-def find_tests_for_modified_files(modified_files, conf, id_set):
+def find_tests_and_content_packs_for_modified_files(modified_files, conf, id_set):
     script_names = set([])
     playbook_names = set([])
     integration_ids = set([])
 
-    tests_set, catched_scripts, catched_playbooks = collect_changed_ids(integration_ids, playbook_names,
-                                                                        script_names, modified_files, id_set)
-    test_ids, missing_ids, caught_missing_test = collect_tests(script_names, playbook_names, integration_ids,
-                                                               catched_scripts, catched_playbooks, tests_set, id_set,
-                                                               conf)
+    tests_set, catched_scripts, catched_playbooks, packs_to_install = collect_changed_ids(
+        integration_ids, playbook_names, script_names, modified_files, id_set)
+
+    test_ids, missing_ids, caught_missing_test, test_packs_to_install = collect_tests_and_content_packs(
+        script_names, playbook_names, integration_ids, catched_scripts, catched_playbooks, tests_set, id_set, conf)
+
+    packs_to_install.update(test_packs_to_install)
```

Comment:
why don't you just pass `packs_to_install` to `collect_tests_and_content_packs`

Reply:
hmm why would i do that? it makes more sense to merge the two here in my opinion